spec/javascripts/admin/modules/paste_govspeak.spec.js

```diff
@@ -0,0 +1,55 @@
+describe('GOVUKAdmin.Modules.PasteGovspeak', function () {
+  var textarea, pastedGovspeak
+
+  function createHtmlPasteEvent (html = null) {
+    const event = new window.Event('paste')
+    event.clipboardData = {
+      getData: (type) => {
+        if (type === 'text/html') {
+          return html
+        }
+      }
+    }
+
+    return event
+  }
+
+  beforeEach(function () {
+    textarea = document.createElement('textarea')
+
+    pastedGovspeak = new GOVUKAdmin.Modules.PasteGovspeak()
+    pastedGovspeak.start($(textarea))
+  })
+
+  afterEach(function () {
+    textarea.remove()
+  })
+
+  it('should convert simpled formatted content to GovSpeak', function () {
```

Comment:
```suggestion
  it('respond to a paste event by converting the HTML to Govspeak', function () {
```

note: we refer to it as Govspeak in documentation rather than GovSpeak e.g: https://www.gov.uk/guidance/how-to-publish-on-gov-uk/markdown

Reply:
done